doc/modules/naive_bayes.rst

```diff
@@ -124,7 +124,7 @@ version of maximum likelihood, i.e. relative frequency counting:
 where :math:`N_{yi} = \sum_{x \in T} x_i` is
 the number of times feature :math:`i` appears in a sample of class :math:`y`
 in the training set :math:`T`,
-and :math:`N_{y} = \sum_{i=1}^{|T|} N_{yi}` is the total count of
+and :math:`N_{y} = \sum_{i=1}^{n} N_{yi}` is the total count of
```

Comment:
Gah... yeah so this sum is actually over the features. Maybe I should read properly before commenting @TropComplique  you're right... This is a bit odd because we usually define ``y_i`` as the label of sample ``i``. The fix is good, but I might even suggest rewriting the whole thing with a more standard sklearn notation.  I'm not sure if we have actual naming rules for one-letter names for ``n_features`` and ``n_samples``. I think we sometimes use ``n`` and ``N`` and sometimes use ``p`` and ``n`` :(